cms/src/components/content/ContentOverview.spec.ts

```diff
@@ -126,6 +126,32 @@ describe("ContentOverview.vue", () => {
         });
     });
 
+    it("should display filter options and inputs", async () => {
+        const wrapper = mount(ContentOverview, {
+            global: {
+                plugins: [createTestingPinia()],
+            },
+            props: {
+                docType: DocType.Post,
+            },
+        });
+
+        await waitForExpect(async () => {
+            const showFilterOptionsBtn = wrapper.find('[data-test="show-filter-options-btn"]');
+            expect(showFilterOptionsBtn.exists()).toBe(true);
+            await showFilterOptionsBtn.trigger("click");
+
+            const filterOptions = wrapper.find('[data-test="filter-options"]');
+            expect(filterOptions.exists()).toBe(true);
```

Comment:
Similar to the other PR, I don't think testing that the wrapping div exists is totally necessary.

Reply:
Discussed on Discord